addons/account/tests/test_account_journal_dashboard.py

```diff
@@ -89,12 +89,12 @@ def test_customer_invoice_dashboard(self):
         self.assertEqual(dashboard_data['number_draft'], 0)
         self.assertIn('0.00', dashboard_data['sum_draft'])
 
-        self.assertEqual(dashboard_data['number_waiting'], 2)
-        self.assertIn('78.42', dashboard_data['sum_waiting'])
+        self.assertEqual(dashboard_data['number_waiting'], 1)
+        self.assertIn('81.72', dashboard_data['sum_waiting'])
```

Comment:
Shouldn't we rather make a partial payment of the invoice, here, instead of the refund, since it's now ignored and we already check that above ?

Reply:
I let you have a look at what I do now before resolving the comment.

As refunds and payments on refunds are ignored by the dashboard which only shows the residual of invoices, after testing the initial case, i test that making a partial payment on the invoice reduces the amount shown on the dashboard without changing the number of moves concerned before anyting else.